### PR TITLE
Update YANG models

### PR DESCRIPTION
**Pull request by aguoietf**
Summary of the updates:
- Added Esther, Julien, Gert, Roberto as contributors
- Updated the type for delta-power to ratio-in-db
- Updated YANG trees, txt and xml 
- Moved dependent YANG files to the yang-dep folder

---

Pending merge of https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext-RFC9093-bis/pull/82

### CODE REVIEW

**Review by sergiobelotti — APPROVED** [review]

**Review by italobusi — APPROVED** [review]

**Comment by italobusi** [comment]
2023-12-21 Call

The PR is ready to be merged as soon as https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext-RFC9093-bis/pull/82 is merged

**Review by ggalimba56 — APPROVED** [review]

File: xml-txt/draft-ietf-ccamp-wdm-tunnel-yang.txt [hunk 1]
```diff
@@ -0,0 +1,3640 @@
...
+
+   *  Global transponder/transceiver configuration constraints, e.g.,
+      operational modes, tuning constraints of the transceiver
+
+   *  Global optical performance constraints, e.g. generalized Signal-
+      to-noise (G-SNR) margin of a feasible optical path
+
+   *  Path-scope WDM layer constraints, e.g. identities of transceivers
+      assigned to the primary or secondary path
+
```
[the remark this answers is not in the document]
Comment (aguoietf) [reply xml-txt/draft-ietf-ccamp-wdm-tunnel-yang.txt:181]: Yes they are covered by the TE tunnel model from which wdm tunnel model augments

File: xml-txt/draft-ietf-ccamp-wdm-tunnel-yang.txt [hunk 2]
```diff
@@ -0,0 +1,3640 @@
...
+                         Figure 1: Topology Example
+
+   To configure an end-to-end WDM tunnel to interconnect transponders A
+   and E, first of all we have to populate the flexi-grid topology YANG
+   model with all elements in the network:
+
+   *  We define the transponders within nodes A and E as tunnel
+      termination points (TTPs) and provide their internal local link
+      connectivity towards the node interfaces.  We also provide nodes A
+      and B identifiers, addresses and interfaces.
```
[the remark this answers is not in the document]
Comment (aguoietf) [reply xml-txt/draft-ietf-ccamp-wdm-tunnel-yang.txt:263]: ok updated

File: xml-txt/draft-ietf-ccamp-wdm-tunnel-yang.txt [hunk 3]
```diff
@@ -0,0 +1,3640 @@
...
+               /te:explicit-route-objects-always
+               /te:route-object-exclude-always/te:type/te:label
+               /te:label-hop/te:te-label/te:technology:
+       +--:(wdm)
+          +--rw (grid-type)?
+             +--:(fixed-dwdm)
+             |  +--rw (fixed-single-or-super-channel)?
+             |     +--:(single)
+             |     |  +--rw dwdm-n?               l0-types:dwdm-n
+             |     +--:(multi)
```
[the remark this answers is not in the document]
Comment (aguoietf) [reply xml-txt/draft-ietf-ccamp-wdm-tunnel-yang.txt:601]: I think this information about channel spacing is already defined in the label-step augments for fixed DWDM:
wson-dwdm-channel-spacing is an identityref which contains ITU-T G.694.1 defined DWDM grids.
 
~~~
  augment /te:te/te:tunnels/te:tunnel/te:primary-paths
            /te:primary-path/te:path-in-segment/te:label-restrictions
            /te:label-restriction/te:label-step/te:technology:
    +--:(wdm)
       +--rw (l0-grid-type)?
          +--:(fixed-dwdm)
          |  +--rw wson-dwdm-channel-spacing?    identityref
          +--:(cwdm)
          |  +--rw wson-cwdm-channel-spacing?    identityref
          +--:(flexi-grid)
             x--rw flexi-grid-channel-spacing?   identityref
             +--rw flexi-ncfg?                   identityref
             +--rw flexi-n-step?                 uint8
~~~

**Pull request merged**